react/features/notifications/functions.js

```diff
@@ -15,3 +17,12 @@ export function areThereNotifications(stateful: Object | Function) {
 
     return enabled && notifications.length > 0;
 }
+
+/**
+ * Tells wether join/leave notifications are enabled in interface_config.
+ *
+ * @returns {boolean}
+ */
+export function joinLeaveNotificationsDisabled() {
+    return interfaceConfig.DISABLE_JOIN_LEAVE_NOTIFICATIONS;
```

Comment:
Maybe add a `Boolean()` around it, sincce it might be undefined.

Reply:
Also, check if `interfaceConfig` itself is undefined. This will be the case on mobile.